docs/tutorials/webcam-line-follower-robot.md

```diff
@@ -381,7 +381,7 @@ This will give it a wider field of view so it takes longer for the line to go ou
 
 Things to try:
 
-- Add a `saturation_cutoff_pct` and/or a `value_cutoff_percent` [(documented here)](/services/vision/#color-detector-parameters) to your Vision Service parameters.
+- Add a `saturation_cutoff_pct` and/or a `value_cutoff_percent` [(documented here)](/services/vision/detection#color-detector) to your Vision Service parameters.
```

Comment:
```suggestion
- Add a `saturation_cutoff_pct` and/or a `value_cutoff_percent` [(documented here)](/services/vision/detection#`color-detector`) to your Vision Service parameters.
```

This may fix HTML

Reply:
It seems like it wants not backticks but rather a slash for some reason (this is how it is in the rendering):
```suggestion
- Add a `saturation_cutoff_pct` and/or a `value_cutoff_percent` [(documented here)](/services/vision/detection/#color-detector) to your Vision Service parameters.
```